example/90-shoot-local.yaml

```diff
@@ -76,7 +76,7 @@ spec:
   #     SomeKubernetesFeature: true
   dns:
     provider: unmanaged
-    domain: <minikube-ip>.nip.io
+    domain: <local-kubernetes-ip>.nip.io
```

Comment:
Please change this in `hack/templates/resources/90-shoot.yaml.tpl` because these examples are rendered by running `hack/generate-examples`.